Full-featured English website, second try

mgrabovsky opened the pull request:
What remains to be done:
- [x] Translate captions for infographics.
- [x] Translate and check localisation texts in [`_data/lang.yml`](/faktaoklimatu/web-en/blob/preview-full-fledged/_data/lang.yml).
- [x] Translate the front page ([`_pages/index.md`](/faktaoklimatu/web-en/blob/preview-full-fledged/collections/_pages/index.md)).
- [x] Translate the “About us” page ([`_pages/about.html`](/faktaoklimatu/web-en/blob/preview-full-fledged/collections/_pages/about.html)).
- [x] Translate the “Members” page ([`_data/organization-members.yml`](/faktaoklimatu/web-en/blob/preview-full-fledged/_data/organization-members.yml)).
- [x] Translate the “How to use” page ([`_pages/how-to-use.html`](/faktaoklimatu/web-en/blob/preview-full-fledged/collections/_pages/how-to-use.html)).
- [x] Translate the “Privacy policy” page ([`_pages/gdpr.html`](/faktaoklimatu/web-en/blob/preview-full-fledged/collections/_pages/gdpr.html)).
- [x] Translate texts (headings, short names and descriptions) for individual tags (topics and scopes) ([`_data/tags.yml`](/faktaoklimatu/web-en/blob/preview-full-fledged/_data/tags.yml)).
- [x] Translate links to datasets and original sources in sidebar. @katkolouchova 
- [x] Decide on placeholder for accompanying text.
- [x] Include "New stuff" section on the front page. @mgrabovsky
- [x] Fix CSS on the front page. @mgrabovsky
- [x] Include CZ electricity transition studies. @mgrabovsky
- [x] Translate licensing terms.
- [x] Move content into `collections/` directory, see faktaoklimatu/web-core#166. @mgrabovsky 

[Preview link](https://en-facts-on-climate--preview-full-fledged-39cc9hqn.web.app)

mgrabovsky committed:
Update web-core

mgrabovsky committed:
Add a placeholder topic

mgrabovsky committed:
Barebones front page and data files

mgrabovsky committed:
Use a special name for container

mgrabovsky committed:
Initial graphics import

Jirka-tisnov committed:
Přeložení caption do EN

mgrabovsky committed:
Adjust wording in placeholder

mgrabovsky committed:
Move all collections into one directory

See faktaoklimatu/web-core#166 for details.

mgrabovsky committed:
Add Czech grid transition studies

Jirka-tisnov committed:
Translate captions to English

katkolouch committed:
Translate links to datasets and original sources

Jirka-tisnov committed:
Dokončení překladu do EN

@katkolouchova Nejsem si jistý, co znamená "How to use correctly?" Ubral jsem otazník, ale pořád to není OK, chybí předmět věty. A taky si nejsem jistý větou "Our surveys and..." Nevím, co se myslí tím "surveys", ale tuším v tom nějakou habaďůru :-D

katkolouch commented:
@Jirka-tisnov k _How to use correctly_, jde o překlad níže uvedeného (Jak správně použít), takže lepší by asi bylo něco jako _How to use our materials correctly?_

<img width="275" alt="image" src="https://user-images.githubusercontent.com/49321752/178021974-de52a4b1-4b91-49ed-9cde-c9b750b8cf68.png">

Ad surveys – myslím, že tím dotyčný myslel rešerše, vím, že jsme to kdysi řešili a debatovali nad tím:

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/49321752/178022806-df798061-2c2d-4b64-a642-1bdf7c9e460d.png">

Jirka-tisnov commented:
@katkolouchova Myslel jsem si to, ale podle mě je to zavádějící. Řešil jsem to i s několika rodilými mluvčími. V některém kontextu se dá použít "literature review", případně "syntesis". Mně osobně by se líbilo něco jako "synthesis of relevant research data". Případně "summary of...". Je to sice celkem dlouhé, ale mnohem přesnější, navíc to má výhodu v tom, že se to hodí pro aktivitu "řešerše" i výsledný výstup (produkt) "rešerše". "Survey" je prostě něco jiného, co v FoK neděláme.

Jirka-tisnov commented:
@katkolouchova Nebo možná ještě takto: "review & summary of relevant sources".

katkolouch commented:
@Jirka-tisnov Review & summary of relevant sources se mi z navženého líbí nejvíc. Akorát to ještě neodpovídá úplně pro všechny rešerše, co máme. Třeba [rešerše emisních závazků](https://faktaoklimatu.cz/studie/2021-reserse-zavazky-statu) je blíže výtahu, tedy spíše něco jako summary nebo digest. Nakonec zdroje tam taky přidáváme, tak by se to pod tvůj návrh asi taky vešlo.

Jen to pak v titulcích a popiscích bude možná moc dlouhé:
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/49321752/178223027-92b542b7-d65a-468f-a40d-88937bd4aa99.png">

katkolouch commented:
@Jirka-tisnov Review & summary of relevant sources se mi z navženého líbí nejvíc. Akorát to ještě neodpovídá úplně pro všechny rešerše, co máme. Třeba [rešerše emisních závazků](https://faktaoklimatu.cz/studie/2021-reserse-zavazky-statu) je blíže výtahu, tedy spíše něco jako summary nebo digest. Nakonec zdroje tam taky přidáváme, tak by se to pod tvůj návrh asi taky vešlo.

Jen to pak v titulcích a popiscích bude možná moc dlouhé:
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/49321752/178223027-92b542b7-d65a-468f-a40d-88937bd4aa99.png">

Jirka-tisnov commented:
@katkolouchova Jo jo. Pravděpodobně skončíme u toho, že to budeme překládat situačně. Tedy budeme mít více variant, které využijeme podle kontextu. Občas překladatelům nic jiného nezbývá, když není vhodný ekvivalent. Lepší tak, než nějaké krátké a zavádějící řešení jako třeba "survey".

katkolouch commented:
@Jirka-tisnov Ony jsou tyto typy produktů v nadpisech a titulcích dlaždic už předdefinované a přiřazují se podle typu stránky. Takže kdybychom to chtěli dělat situačně museli bychom upravit proces. V textech můžeme označení lehce upravovat podle situace. Takže něco výstižného pro případy, kdy se přiřazuje typ automaticky by se hodilo. Ale zatím naštěstí žádnou rešerši přeloženou nemáme :)

Jirka-tisnov commented:
@katkolouchova Aha, hmm, tak to je zatrachtilá věc. No, co se dá dělat. Já nic lepšího nemám.

katkolouch committed:
About us update

Jirka-tisnov committed:
Přeložení "About" do EN

katkolouch committed:
Fix slug

katkolouch committed:
Add team photos

katkolouch committed:
Remove summaries

Jirka-tisnov committed:
Translate Privacy policy

katkolouch committed:
Remove News page

Jirka-tisnov committed:
Translate front page

katkolouch committed:
Tidying up

mgrabovsky committed:
Update localisation strings

- Rename "scopes" to "regions".
- Localise "back to homepage" button on tag pages.

mgrabovsky committed:
Finish front page

Finish "New materials" section and spacing on the front page.

mgrabovsky committed:
Update publication dates; write CO₂ properly

mgrabovsky committed:
Use "Fakta o klimatu" consistently

mgrabovsky committed:
Tidying up

mgrabovsky committed:
Translate licensing terms

mgrabovsky commented:
Weakly depends on faktaoklimatu/web-core#184 to land.

jankrcal reviewed (changes requested):
My first comment relates to the placeholder-foo.md files: why are those needed?

That seems quite a hack that calls for fixes in web-core so that those collections do not have to exist / can stay empty. WDYT?

jankrcal committed:
Delete unused team member's photo

jankrcal commented on collections/_infographics/climate/climate-change-scheme.md:
```diff
@@ -0,0 +1,15 @@
+caption:       "Climate change is not just a change of temperature – this umbrella term covers a number of interrelated phenomena. The change of one factor (e.g. higher concentrations of CO2 in the atmosphere) results in a long causal chain."
+---
+
+_Our comments on this infographic providing relevant context and useful links are [only available in Czech](https://faktaoklimatu.cz/infografiky/schema-klimaticke-zmeny) at the moment. Please check back later for the English translation._
```
Why not using a local include with this text (with an param `url`) for all the content files?

mgrabovsky replied:
Good idea, I'll use an include instead.

jankrcal commented on collections/_pages/gdpr.md:
```diff
@@ -0,0 +1,52 @@
+
+Address: Cyrilská 7, 602 00 Brno, ID: 095 55 587
+
+Contact person: Anna Macková 
```
Is this still true?
(I know she's listed on the Czech version as well)

Do we have to mention a contact person?

mgrabovsky replied:
I can't tell if the name is still up to date, but I would keep it consistent with the Czech version. We can try to find out if a different name should be there.

Yes, [contact info is mandatory](https://gdpr.eu/privacy-notice/):
> If an organization is collecting information from an individual directly, it must include the following information in its privacy notice:
> * The identity and contact details of the organization, its representative, and its Data Protection Officer
> […]
> If an organization obtains your data indirectly (via another organization) its privacy notice must provide all the same information

jankrcal replied:
Ack

jankrcal commented on collections/_pages/how-to-use.md:
```diff
@@ -0,0 +1,76 @@
+
+You can also access the underlying data used in our infographics.
+
+* By clicking "Our dataset" next to an infographic, you can access the page where you will find the data that we were using when working on this particular infographic (incl. a description of this data and other useful links).
```
We don't publish any dataset in EN, maybe we can comment this bullet point out?

mgrabovsky replied:
Removed mentions of datasets.

jankrcal commented on _data/tags.yml:
```diff
@@ -0,0 +1,69 @@
+
+- id: cz
+  name-short: CZ
+  name-long: the Czech Republic
```
I am not a language professional but I'd rather omit the "the" from name-long of the scope tags. To me, it looks silly on the topics page. Was this approved by or even pushed for by Jirka?

jankrcal committed:
Changes in index.md mobile styles and applying expandable blocks

jankrcal commented:
For the record, I've made a few minor changes. I'll wait for you feedback before making any further adjustments.

mgrabovsky commented:
I'll get back to this later today or tomorrow.

mgrabovsky committed:
Remove definite articles from tag titles

mgrabovsky committed:
Remove mentions of datasets for now

mgrabovsky committed:
Factor placeholder out to an include

mgrabovsky committed:
More localisation following web-core changes

mgrabovsky committed:
Remove placeholder files

mgrabovsky committed:
Update navigation menu

mgrabovsky commented:
@jankrcal I've incorporated your suggestions. Regarding the placeholder files, I feel like they were required initially because the build failed without them, but it seems to go through now, so I've deleted them.

I've also updated to the latest web-core, with the new footer and main nav. Unfortunately, this breaks the layout of the topics accordion a bit:

![Screenshot 2022-08-18 at 18-24-48 Fakta o klimatu](https://user-images.githubusercontent.com/134321/185467944-424837eb-f86e-4192-a6fa-ce62c24ad0f5.png)

jankrcal commented:
@mgrabovsky My bad, I've removed that CSS in https://github.com/faktaoklimatu/web-core/pull/183, reverting in https://github.com/faktaoklimatu/web-core/pull/199.

jankrcal commented:
And thanks for updating the footer! I thought that will end up on my plate :)

jankrcal reviewed (approved):
Looks good to me, thanks for the great work!

(please first merge the web-core PR, also check my minor changes whether you're fine with them)

jankrcal commented:
We could consider updating the homepage similarly to the Czech version but that's definitely for a separate PR.

Later on, I plan to port the Slovak and English sites to use the new topic pages (to reduce technical debt, without any of the elaborate features used on the Czech site to keep the amount of work minimal).

jankrcal committed:
Add support-us button to the footer (mobile-only)

jankrcal committed:
Semicolon in Makefile (fixing make 4.3)

jankrcal commented:
@mgrabovsky I've sneaked in a tiny fix of https://github.com/faktaoklimatu/web-cz/issues/1102, could you please test it does not break anything for you before merging this PR?

mgrabovsky commented:
@jankrcal I will. Thanks for the review and your changes. I will take a final look later today.

mgrabovsky committed:
Update web-core
